libraries/engage/favorites/components/List/List.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Fragment } from 'react';
```

Comment:
![Bildschirmfoto 2022-05-06 um 17 01 57](https://user-images.githubusercontent.com/13087792/167159679-801e36c9-0051-4d30-aad7-b40c7c5ceca7.png)
There is much more whitespace around and inside the items compared with the original implementation. Guess you could just completely migrate the components instead of mixing old with new.

Additionally for me it always takes a while till the space of a removed item is released and the list shrinks.

Reply:
done